torch/csrc/utils/crash_handler.cpp

```diff
@@ -36,22 +36,49 @@ void _enable_minidump_collection(const std::string& dir) {
       -1);
 }
 
+void _disable_minidump_collection() {
+  handler = nullptr;
```

Comment:
Not sure whether it's any better, but at least to me it gives a context, that handler is probably a managed pointer.
```suggestion
  handler.reset();
```